patches/ui-base-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
+index 3452178f0db87..7b9ede7f64974 100644
+--- a/ui/base/BUILD.gn
++++ b/ui/base/BUILD.gn
+@@ -552,6 +552,7 @@ component("base") {
+       "//ui/events",
+       "//ui/events/devices",
+     ]
++    import("//brave/chromium_src/ui/base/sources.gni") deps += brave_chromium_src_ui_base_deps
```

Comment:
this is inside `use_blink` which is wrong and it fails on ios. Also the correct dep to use is `//brave/components/resources:strings`